streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -529,11 +531,13 @@ public void run() {
             cleanRun = true;
         } catch (final KafkaException e) {
             // just re-throw the exception as it should be logged already
+            // TODO: this is not true since task.update() call may throw ProcessorStateException
+            //       which is a KafkaException but it never get logged.
             throw e;
         } catch (final Exception e) {
             // we have caught all Kafka related exceptions, and other runtime exceptions
             // should be due to user application errors
-            log.error("{} Streams application error during processing: ", logPrefix, e);
+            log.error("{} Streams application error during processing", logPrefix, e);
```

Comment:
Yes, the stack trace starts at the next line. But you example shows

```
suspending:(org.apache.kafka.streams.processor.internals.StreamThread:1180)
```

The `:` we are discussion about is the one after "suspending", right? Thus, IMHO a `:[BLANK]` would be a nice delimiter from the suffix showing the original line where the exception is throws. Having say this, a single `[BLANK]` without a `:` would also do, but nothing in between would not render nicely IMHO: eg. `[...]suspending(org.apache[...])`